src/Modules/Profie/index.js

```diff
@@ -185,6 +185,7 @@ class Profile extends Component {
                   placeholder={editingUserDetails ? 'write your bio' : null}
                   readOnly={!editingUserDetails}
                   name="bio"
+                  spellcheck="false"
```

Comment:
What's this for?

Reply:
![image](https://user-images.githubusercontent.com/29652551/66122220-c91ab200-e5fc-11e9-9853-9e4250f2926f.png)

this one is for disabling the default property **spellCheck** of textarea.

Reply:
I'm not sure is it a good practice or not.